JavatrixTest.java

```diff
@@ -570,6 +570,63 @@ public void minusNewObj()
     }
 
     /**
+     * Test timesEquals function, valid values.
+     */
+    @Test
+    public void timesEqualsValid()
+    {
+        m = 2;
+        n = 3;
+        double scalar = 3.3;
+        double[] a = {1, 2, 3};
+        double[] b = {4, 5, 6};
+        matrix = new double[m][n];
+        matrix[0] = a;
+        matrix[1] = b;
+
+    	actual = new Javatrix(matrix);
```

Comment:
Um, Why? None of them are in anything but the test it's self. Also which line exactly?